tls.c

```diff
@@ -160,6 +164,8 @@ static bool load_server_certificates(char **errmsg) {
         free(error_msg);
     } else {
         *errmsg = error_msg;
+        error_msg += (err_msg_size >= errmax ? errmax - 1: err_msg_size);
+        snprintf(error_msg, CRLF_NULLCHAR_LEN, "\r\n");
```

Comment:
shouldn't this be errmax - CLRF_NULLCHAR_LEN instead of - 1?

Reply:
sprintf only appends '\0', there's no CRLF.

Reply:
unless I'm missreading, you're offsetting the error_msg variable to errmax - 1, but then snprintf'ing 2 chars + null byte? so that'd be two bytes past the allocation if errmax were hit?

Reply:
errmax is CRLF_NULLCHAR_LEN bytes shorter than the size of error_msg.

Reply:
After squinting again, I see I misread a - for a +. Thanks!